support/commercial.rst

```diff
@@ -5,6 +5,8 @@ Paid Support and Services
 
 Here are some commercial service and support providers for borgbackup:
 
+- `Lima-Labs <https://storage.lima-labs.com/>`_ - remote live file systems with native borg support
```

Comment:
move this to the end of the list please.

also, when users look at this list, guess they will ask themselves what's the difference between:
- `remote live file system` and `cloud storage platform`
- `native borg support` and `borg support`

In general, i want the least marketing phrasing here, just telling what they offer.

Reply:
maybe we should just say "storage with borg support".

Reply:
I agree with you. Fixed

Reply:
ok, thanks!